clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -362,7 +362,7 @@ private void transitionTo(MemberState nextState) {
             metricsManager.recordRebalanceStarted(time.milliseconds());
         }
 
-        log.trace("Member {} with epoch {} transitioned from {} to {}.", memberId, memberEpoch, state, nextState);
+        log.info("Member {} with epoch {} transitioned from {} to {}.", memberId, memberEpoch, state, nextState);
```

Comment:
This one seems like it would make for a noisy log. What about `DEBUG` as a compromise?

Reply:
While troubleshooting different scenarios on the stress tests we're running, we always end up finding ourselves struggling to understand the member state just because we don't have this log info handy, so the intention was to move it up at least on this Preview stage to easily track the state machine. This should really only come out on events that we do care about (joining, leaving, reconciling, errors), but if on practice we see it ends up generating more noisy than the value it has we'll lower it down then. Makes sense?

Reply:
OK, yes, that totally makes sense 👍 Plus, when community members file issues against KIP-848 client work, it's better to have those logs included by default. Good call 😄